baseapp/abci.go

```diff
@@ -757,6 +758,24 @@ func handleQueryApp(app *BaseApp, path []string, req abci.RequestQuery) abci.Res
 				Value:     []byte(app.version),
 			}
 
+		case "snapshots":
+
+			response := app.ListSnapshots(abci.RequestListSnapshots{})
+
+			var responseValue []byte
+
+			responseValue, err := json.Marshal(response)
+
+			if err != nil {
+				responseValue = []byte(err.Error())
```

Comment:
Why wrap the error like this instead of returning the error itself as we do in other cases, i.e.

```go
return sdkerrors.QueryResult(sdkerrors.Wrap(err, ...)
```

Reply:
This was fixed